tests/Kernel/fixtures/rendering.yml

```diff
@@ -2331,7 +2331,12 @@
     '#id': text_featured_media
     '#fields':
       title: "Heading"
-      video: "<iframe id=\"videoplayerI-181645\" src=\"//ec.europa.eu/avservices/play.cfm?ref=I-181645\"></iframe>"
+      video:
```

Comment:
Since we can't provide the raw video anymore, we should update the documentation in text_featured_media.ui_patterns.yml to clarify we expect a render array now.